### PR TITLE
Add MulticoreTSNE

### PR DESCRIPTION
**Pull request by grst**
> This is a multicore modification of Barnes-Hut t-SNE by L. Van der Maaten with python and Torch CFFI-based wrappers. This code also works faster than sklearn.TSNE on 1 core.

https://github.com/DmitryUlyanov/Multicore-TSNE

### CODE REVIEW

**Comment by grst** [comment]
@conda-forge/core, please review

File: recipes/multicore-tsne/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    This is a multicore modification of Barnes-Hut t-SNE by L. Van der Maaten
+    with python and Torch CFFI-based wrappers.
+    This code also works faster than sklearn.TSNE on 1 core.
+  dev_url: https://github.com/DmitryUlyanov/Multicore-TSNE
+
+extra:
+  recipe-maintainers:
+    - DmitryUlyanov
```
Comment (grst) [inline recipes/multicore-tsne/meta.yaml]: @DmitryUlyanov, are you ok being added as a maintainer for the conda package?

**Comment by grst** [comment]
@scopatz, good to go now?

**Comment by scopatz** [comment]
Thanks @grst!

**Pull request merged**